app/javascript/components/student/tracks-list/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export function Header({ data, query = {} }) {
+  const isFiltering = query.criteria || (query.tags && query.tags.length !== 0)
```

Comment:
Just a tiny suggestion:

```suggestion
  const isFiltering = query.criteria || (query.tags && query.tags.length > 0)
```

I feel that `> 0` is slightly easier to read than `!== 0`.

Reply:
Sounds good!